packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/dataservice/ModelController.java

```diff
@@ -586,6 +586,7 @@ ResponseEntity<List<ModelConfiguration>> getModelConfigurationsForModelId(
 		@RequestParam(value = "page-size", required = false, defaultValue = "100") final int pageSize,
 		@RequestParam(name = "project-id", required = false) final UUID projectId
 	) {
+		@SuppressWarnings("unused")
```

Comment:
Rather than suppressing we can just get rid of `final Schema.Permission permission = `